ajax/libs/rpg-awesome

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Let's create a folder which named `rpg-awesome` and move this file to `ajax/libs/rpg-awesome/package.json`.
As your commit message show: `via single package.json`.

Many thanks.